### DISCUSSION TITLE
Alexa sensor

### DISCUSSION THREAD

**Discussion by ster1um** [indomus/forum#918] [GENERALE - Discussioni generiche]
Esiste qualche limitazione all'esposizione dei sensori di umidità con l'integrazione di alexa?

Con Google Assistant e HomeKit vengono rilevati tutti i sensori temperatura umidità ed invece con alexa mi espone solo la temperatura.
Ho provato a forzare inserendo le entità in configurazione include_entities senza succeso...

**Comment by Scialla** [comment]
è normale, su alexa niente umidità (per ora)

**Reply by ster1um** [reply]
> 
> 
> è normale, su alexa niente umidità (per ora)

ci stavo perdendo la testa con l'esposizione di sti sensori... morta##i loro😉
